modules/setup/config/__init__.py

```diff
@@ -72,7 +74,9 @@ async def create(args):
         config_module.stoploss = config["stoploss"]
         config_module.stoploss_type = config["stoploss-type"]
         config_module.max_open_trades = config["max-open-trades"]
+        config_module.exposure_per_trade = config["exposure-per-trade"] / 100.
         config_module.plots = config["plots"]
+        config_module.tearsheet = config["tearsheet"]
```

Comment:
Ah yeah, this is a breaking change. existing configs will struggle on this (hence the failing end-2-end tests).
The following will access safely and provide a default value.
```suggestion
        config_module.exposure_per_trade = config.get("exposure-per-trade", 100.) / 100.
        config_module.plots = config["plots"]
        config_module.tearsheet = config.get("tearsheet", False)
```

Reply:
I'm checking pytest on windows now but should stop us from merging